guides/common/modules/proc_disabling-tftp-for-integration.adoc

```diff
@@ -0,0 +1,42 @@
+[id="disabling-tftp-for-integration"]
+= Disabling TFTP for integration
+
+If you want to manually manage a TFTP service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
+
+[NOTE]
+====
+Disabling TFTP in {Project} does not remove the related backend service on the operating system.
+====
+
+
+.Prerequisites
+
+* You are logged in to the {ProjectWebUI}.
+
+
+.Procedure
+
+. Navigate to *Infrastructure* > *Subnets*.
+
```

Comment:
```suggestion
.Prerequisites
* You are logged in to the {ProjectWebUI}.

.Procedure
. Navigate to *Infrastructure* > *Subnets*.
```
```suggestion
.Procedure
. Navigate to *Infrastructure* > *Subnets*.
. For each subnet that is associated with the TFTP {SmartProxy}:
```
@mmuehlfeldRH Can you please make sure there are no unnecessary blank lines? This here is just an example, it applies to other parts of your PR too.

Reply:
[You are not the only one](https://github.com/theforeman/foreman-documentation/pull/3769#issuecomment-2788878802) :) But yes, if you look at the other content (at least the vast majority that is more or less up-to-date and maintained), there are no blank lines like this. So this is indeed a matter of consistency.

Reply:
I removed the blank lines